packages/core/ts/MobileCore.ts

```diff
@@ -182,7 +182,7 @@ const MobileCore: IMobileCore = {
    *  button or a certain article was viewed.
    *
    *  note: when using the Adobe Analytics extension, calling this API will increment page views
-   * @param  {String?} state containing the name of the state to track
+   * @param  {String?} action containing the name of the action to track
    * @param  {{ string: string }?} contextData containing context data to attach on this hit
```

Comment:
* @param {Record<string, string>} contextData containing context data to attach on this hit